en/trying-out-the-language.md

```diff
@@ -16,16 +16,16 @@ It's here:
 
 ## Trying on your system
 
-For this case, you first need to install One compiler, currently, You are able to compile the One compiler in Linux (both x86_64 and i386 architecture) and macOS, and BSD family, and finally Windows OS.
```

Comment:
For this case, you first need to install One compiler. **Currently you** are able to compile the One compiler **on** Linux (both x86_64 and i386 architecture), **macOS, the BSD family, and finally Windows OS.**